src/app/contextBroker/contextBroker.cpp

```diff
@@ -256,6 +256,7 @@ bool            logDeprecate;
 #define REQ_POOL_SIZE          "size of thread pool for incoming connections"
 #define IN_REQ_PAYLOAD_MAX_SIZE_DESC   "maximum size (in bytes) of the payload of incoming requests"
 #define OUT_REQ_MSG_MAX_SIZE_DESC      "maximum size (in bytes) of outgoing forward and notification request messages"
+#define THRESHOLD_MAX_SIZE_DESC        "maximum threshold limit for notificationQueue"
```

Comment:
It seems you tried to use a CLI configurable threshold but at the end you go for a hardwarired one :)

I think CLI parameter for this would add a needed degree of flexibility. We could use 80% by default if no CLI is provided, but the user could set it to a another value depending of the requirements of its use case (e.g. raise the threshold to 90% or decrease it to 60%).

Reply:
Added in https://github.com/telefonicaid/fiware-orion/pull/4375/commits/51a71bbd8113d91b0c1858d8f1f1aada294a61be